t/LWP/ConsoleLogger/Everywhere.t

```diff
@@ -0,0 +1,54 @@
+use strict;
+use warnings;
+
+use Capture::Tiny 'capture_stderr';
+use Path::Tiny qw( path );
+use Test::FailWarnings;
+use Test::Fatal qw( exception );
+use Test::More;
+use LWP::UserAgent;
+use WWW::Mechanize;
+
+use LWP::ConsoleLogger::Everywhere;
+
+my $foo = 'file://' . path('t/test-data/foo.html')->absolute;
+
+my $lwp  = LWP::UserAgent->new( cookie_jar => {} );
+my $mech = WWW::Mechanize->new( autocheck  => 0 );
+
+foreach my $ua ( $lwp, $mech ) {
+    my $stderr = capture_stderr sub {
+        is(
+            exception {
+                $mech->get($foo);
+            },
+            undef,
+            'code lives'
+        );
+    };
+    ok $stderr, 'there was a dump';
+}
+
+{
+
+    package Foo::Bar;
+
```

Comment:
Why is this required? Is this a test for the feature "works in all modules not just the one where you used it"? If so, why is the previous required?